Cargo.toml

```diff
@@ -26,6 +26,7 @@ features = ["stm32f429", "rt", "usb_fs"]
 targets = ["thumbv7em-none-eabihf"]
 
 [dependencies]
+bxcan = ">=0.3.0"
```

Comment:
bxcan = { version = "0.3.0", optional = true }

and then below "has-can = ["bxcan"]" to make bxcan optional?